config/vufind/EDS.ini

```diff
@@ -323,3 +324,27 @@ AU = None
 ; documentation on options for different adapters.
 ; ttl, Time to Live, i.e. cache entry life time in seconds. 300 seconds by default.
 ;options[ttl] = 300
+
+; This section supports the Databases recommendation module,
+; which displays a list of the results' databases, each linking
+; to that database's individual website.
+[Databases]
+
+; Define the facet hierarcy path leading to each database.
```

Comment:
Typo: hierarcy. (This has to be one of the worst words to type correctly...)

Reply:
Indeed.  Fixed.